modules/ROOT/pages/_partials/link-anypoint-platform-accounts.adoc

```diff
@@ -0,0 +1,83 @@
+. If prompted, enter your credentials for your IdP. +
+If your SSO credentials share an email address with additional profiles in the same organization, you are prompted to select one of the profiles and log in with those credentials to link your profiles. If you are not prompted, your other profiles use a different email address than the SSO login you are logging in to.
++
+. In the *Link your Account* page, select the profile you want to link. +
+You can hover over each profile to see which identity provider it uses.
++
+. Click *Link Accounts*.
+. Enter your credentials for the profile you chose.
++
+You can now view your linked profile in the *Logged In Via* section of your user profile.
+
+// end::profileLinking[]
+
+// tag::profileLinkingContextForAdmins[]
+[[enable-profile-linking]]
+= Enabling Users to Link Anypoint Platform Profiles
+
+As an organization administrator, you can enable users to link multiple Anypoint Platform profiles that use the same email address to log in. 
+
+Anypoint Platform supports:
+
+* Linking an SSO login to a local profile by using credentials (login managed by Anypoint IdP)
+* Linking an SSO login to another SSO login that uses a different IdP
+* Linking an SSO login to multiple other profiles that use different IdPs
+
+Anypoint Platform does not support linking multiple profiles that use the same IdP. Note that only OpenID Connect (OIDC) IdPs are supported.
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+include::partial$include-nav-identity-providers.adoc[]
+
+. In the *Link Multiple Accounts* section, click *Enabled*. 
+. Click *Save Changes*. 
++
+Users in your organization can now link their SSO logins with other IdP profiles that share the same email address.
+
+// end::profileLinkingContextForAdmins[]
+
+// tag::deleteLinkedProfilesAdmin[]
+[[remove-linked-profile-org-admin]]
+= Remove Link From User Account 
+
+Organization administrators can remove a linked profile from a user account by making a `DELETE` call to `/accounts/api/organizations/:orgId/users/:userId/identityProviderProfiles`. For details, refer to the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/[Access Management API documentation]. 
+
+// end::deleteLinkedProfilesAdmin[]
+
+// tag::requireProfileLinkingForAdmins[]
+[[require-profile-linking]]
+= Require Users to Link Anypoint Platform Profiles
+// DO NOT USE THIS SECTION YET - NOT YET SUPPORTED 
+
+If your organization has xref:external-identity-index.adoc#enable-profile-linking[enabled the *Link Multiple Accounts* setting], organization administrators can require all users to link available Anypoint Platform profiles.
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+include::partial$include-nav-identity-providers.adoc[]
+
+. In the *Link Multiple Accounts* section, click *Required*. 
++
```

Comment:
```suggestion
. In the *Link Multiple Accounts* section, click *Required* so that users can't skip the prompt to link their user profiles when logging in to Anypoint Platform. 
```